static/specification.html

```diff
@@ -445,42 +190,52 @@ <h2><dfn>Monetization</dfn> interface</h2>
             attribute EventHandler onprogress;
           };
         </pre>
-        <dl>
-          <dt><dfn>state</dfn> attribute</dt>
-          <dd>
-            A {{ Monetization / state }} attribute exposes the current state of
-            Web Monetization as a {{ MonetizationState }} enum value.
-          </dd>
-          <dt><dfn>onstart</dfn> attribute</dt>
-          <dd>
-            A {{ Monetization / onstart }} attribute is an
-            {{ EventHandler }} for a {{ MonetizationStartEvent }} named
-            "<a>start</a>".
-          </dd>
-          <dt><dfn>onprogress</dfn> attribute</dt>
-          <dd>
-            A {{ Monetization / onprogress }} attribute is an
-            {{ EventHandler }} for an {{ MonetizationProgressEvent }} named
-            "<a>progress</a>".
-          </dd>
-        </dl>
+        <section>
+          <h2>
+            <dfn>state</dfn> attribute
+          </h2>
+          <p>
+            A {{Monetization/state}} attribute exposes the current state of Web
+            Monetization as a {{MonetizationState}} enum value.
+          </p>
+        </section>
+        <section>
+          <h2>
+            <dfn>onstart</dfn> attribute
+          </h2>
+          <p>
+            A {{Monetization/onstart}} attribute is an {{EventHandler}} for a
+            {{MonetizationEvent}} named <a data-link-for=
+            "Monetization">start</a>.
+          </p>
+        </section>
+        <section>
+          <h2>
+            <dfn>onprogress</dfn> attribute
+          </h2>
+          <p>
+            A {{Monetization/onprogress}} attribute is an A
```

Comment:
This reads a bit confusing on the generated document. I think removing this line entirely would help.

`A onprogress attribute is an A onprogress attribute is an EventHandler for an MonetizationEvent named "progress". `

Reply:
Yeah, sorry, something got lost here... fixing.